ui/lib/login.js

```diff
@@ -25,3 +25,35 @@ export function saveToVisitedOrgs(domain, orgName) {
     })
   }
 }
+
+export function currentBaseDomain() {
+  let domain = window.location.host
+  let parts = domain.split('.')
+  if (parts.length > 2) {
+    parts.shift() // remove the org
+    domain = parts.join('.') // join the last two parts of the domain
+  }
+  return domain
+}
```

Comment:
we can maybe change the domain and parts to `const`
and instead of assigning domain to parts.join('.') we can just return parts.join('.')